internal/cli/cmd/clustermesh.go

```diff
@@ -336,3 +346,49 @@ func newCmdExternalWorkloadStatus() *cobra.Command {
 
 	return cmd
 }
+
+func newCmdClusterMeshEnableWithHelm() *cobra.Command {
+	var params = clustermesh.Parameters{
+		Writer: os.Stdout,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "enable",
+		Short: "Enable ClusterMesh ability in a cluster using Helm",
+		Long:  ``,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			params.Namespace = namespace
+			ctx := context.Background()
+			if err := clustermesh.EnableWithHelm(ctx, k8sClient, params); err != nil {
+				fatalf("Unable to enable ClusterMesh: %s", err)
+			}
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&params.ServiceType, "service-type", "LoadBalancer", "Type of Kubernetes service to expose control plane { ClusterIP | LoadBalancer | NodePort }")
```

Comment:
This changes the default value from the old behavior. Did you have any reason you wanted to change the default?

Reply:
I believe I chose `LoadBalancer` here, because some version of the clustermesh setup docs (which I can't seem to find at the moment), specified `--service-type NodePort`, as though LoadBalancer was the default. 

The default does appear to be `NodePort` https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/values.yaml#L2385

I'll change this to `NodePort`.

Reply:
Thanks for your consideration. I would also like to know which of the cilium-cli and Helm defaults we should preserve. I don't know the overall direction of this work, but if we use the old cilium-cli default, the default is an empty string, and cilium-cli does auto-detection depending on the environment (https://github.com/cilium/cilium-cli/blob/main/clustermesh/clustermesh.go#L111-L135). In your new logic, you cut out the auto-detection logic. Is it expected?

Reply:
Yes, the current direction for much of this autodetection logic is to replace it with documentation, as it has been a source of bugs.